cg/services/illumina_services/illumina_metrics_service/bcl_convert_metrics_parser.py

```diff
@@ -14,13 +14,18 @@
     QUALITY_METRICS_FILE_NAME,
 )
 from cg.io.controller import ReadFile
-from cg.services.bcl_convert_metrics_service.models import DemuxMetrics, SequencingQualityMetrics
+from cg.services.illumina_services.illumina_metrics_service.models import (
+    DemuxMetrics,
+    SequencingQualityMetrics,
+)
 from cg.utils.files import get_file_in_directory
 
 LOG = logging.getLogger(__name__)
 
 
-class MetricsParser:
+class BCLConvertMetricsParser:
+    """Parser file for BCLConvert metrics used for illumina flow cells."""
+
     def __init__(
         self,
         bcl_convert_metrics_dir_path: Path,
```

Comment:
can this be `flow_cell_dir`? I really thought this was a path pointing to a metrics directory but is the flow cell directory, right?

Reply:
lets fix that in a different pr
but I think you are right.